src/CoreWlan/Enums.cs

```diff
@@ -92,6 +92,10 @@ public enum CWSecurity : ulong {
 		[Mac (10,15)]
 		Wpa3Transition = 13,
 		Unknown = long.MaxValue,
```

Comment:
This `Unknown` value should stay at the end (since it's using the last `long` value possible)

Reply:
Good point!